src/stream/src/common/table/test_state_table.rs

```diff
@@ -2045,3 +2045,133 @@ async fn test_state_table_iter_prefix_and_sub_range() {
     let res = iter.next().await;
     assert!(res.is_none());
 }
+
+#[tokio::test]
+async fn test_mem_table_spill_in_streaming() {
```

Comment:
This UT tests the case when there are 3 rows inserted into memtable and `try_flush` is triggered in between. This does test whether the flushed memtable is visible to `get` but doesn't test the spill epoch comparison and whether the latest row with the same key can be retrieved.

In addition to this test, I suggest we add a test or extend the current test to test the following case:
1. insert `(k1, text1)` and `try_flush`
2. insert `(k1, text1')` and `try_flush`
3. get `k1` before and after 2's `try_flush` and make sure `text1'` is returned

Reply:
I see, supplied.